### PR TITLE
[TRANSLATION] Add 1_instruction_tuning EN-KO translation

### PR DESCRIPTION
**Pull request by nsbg**
# December 2024 Student Submission

## Module Completed
- [x] Module 1: Instruction Tuning
- [ ] Module 2: Preference Alignment
- [ ] Module 3: Parameter-efficient Fine-tuning
- [ ] Module 4: Evaluation
- [ ] Module 5: Vision-language Models
- [ ] Module 6: Synthetic Datasets
- [ ] Module 7: Inference
- [ ] Module 8: Deployment

## Notebooks Added/Modified
List any notebooks you've added or modified:
- [ ] Added new example in `module_name/students/my_example.ipynb`
- [ ] Modified existing notebook with additional examples
- [x] Added documentation or comments

## Checklist

- [ ] I have read the module materials
- [ ] My code runs without errors
- [ ] I have pushed models and datasets to the huggingface hub
- [ ] My PR is based on the `december_2024` branch

## Questions or Discussion Points
Add any questions you have or points you'd like to discuss:
1.  I've created the folder structure as shown below, and I need feedback on whether I should proceed with this structure or improve it.
```
├── ko                     # I Added  
│   ├── 1_instruction_tuning          
│   ├── 2_preference_alignment
│   └── ...
├── 1_instruction_tuning   # Original directory       
└── 2_preference_alignment 
```

## Additional Notes
Any other information that might be helpful for reviewers:

Once the feedback on the directory structure is complete, I will proceed with the translation of the practice notebook. And so far, none of the labels on the open PR titles seem to exactly match the work I've done, so I didn't add labels like [SUBMISSION] or [MODULE]. What label would be appropriate?

### CODE REVIEW

**Comment by mcdaqc** [comment]
[TRANSLATION] is good. I'm using it.

**Comment by nsbg** [comment]
@burtenshaw okay :)

**Comment by nsbg** [comment]
@burtenshaw I removed the duplicated content and added Korean version notebook files. Check and review please.

**Comment by burtenshaw** [comment]
> @burtenshaw I removed the duplicated content and added Korean version notebook files. Check and review please.

Sorry @nsbg, I wasn't clear. I'd like for there to only be Korean versions of files in `ko/` and for all English files like `ko/2_preference_alignment/orpo.md` to be deleted from `ko/`. The English course can stay at `/`

Duplicate files will make searching the repo difficult. We can of course introduced these files back once we've translated them.

Once we have a single complete translation, we can also move the English course to `en/`.

File: 1_instruction_tuning/README.md
Comment (burtenshaw) [inline 1_instruction_tuning/README.md:1]: Files like this should remain in place until the translation is complete. 

In a separate PR, we will move the english course from `/` to `en/`.

File: ko/1_instruction_tuning/README.md
Comment (burtenshaw) [inline ko/1_instruction_tuning/README.md:1]: These files are in the correct location.

File: ko/2_preference_alignment/README.md
Comment (burtenshaw) [inline ko/2_preference_alignment/README.md:1]: Can english files like this be removed from `ko/`.

**Comment by nsbg** [comment]
@burtenshaw I kept only the folders with completed Korean translations and deleted all the untranslated content. But I’m not sure if this matches what you meant exactly. Let me know if I misunderstood.

**Comment by burtenshaw** [comment]
Thanks. That's good. Could you now undelete the English course files.

**Review by burtenshaw — APPROVED** [review]
I've undeleted the english course. LGTM!

As you translate more modules, just add them to `ko/`.  Thank you!

**Pull request merged**